recipes/arrow-c-glib/meta.yaml

```
@@ -0,0 +1,70 @@
+{% set name = "arrow-c-glib" %}
+{% set version = "8.0.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://dist.apache.org/repos/dist/release/arrow/arrow-{{ version }}/apache-arrow-{{ version }}.tar.gz
+  sha256: ad9a05705117c989c116bae9ac70492fe015050e1b80fb0e38fde4b5d863aaa3
+
+build:
+  number: 0
+  skip: true  # [win]
+
```

```suggestion
  skip: true  # [win]
  run_exports:
    - {{ pin_subpackage('arrow-c-glib') }}
```
Should be something *like* this, but with your ABI compatibility guarantees in mind.